### DISCUSSION TITLE
CefBrowser goes white blank after 3-4 minutes.

### DISCUSSION THREAD

**Discussion by Sehlor** [cefsharp/CefSharp#4367] [General]
- **What version of the product are you using?**
From Nuget Latest Version

x86
	
- **What version of .Net?**
    .Net 4.7.3
    
- **On what operating system?**
    Win11

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    WinForms
    
- **What steps will reproduce the problem?**
    - Whenever I embed CefBrowser into my WinForms app, it opens an loads without an issue, but after 3-4 minutes, page goes blank.
    - Does the problem reproduce using the [MinimalExample](https://github.com/cefsharp/CefSharp.MinimalExample)?
    
- **What is the expected output? What do you see instead?**
- I expect to CefBrowser to keep working, it blanks out white instead.

- **Please provide any additional information below.**
    Not additional information, it doesn't throw any exceptions or anything.

**Comment by amaitland** [comment]
> * **What version of the product are you using?**
>   From Nuget Latest Version

What is the **exact** version you are using?

> * Does the problem reproduce using the [MinimalExample](https://github.com/cefsharp/CefSharp.MinimalExample)?

Does the problem reproduce with the `MinimalExample`???

> * Not additional information, it doesn't throw any exceptions or anything.

Did you check the log file? See https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

The behaviour you are describing sounds like the render process was terminated, possibly out of memory based on the time. 

As you are using `x86` make sure your application is [LargeAddressAware](https://github.com/cefsharp/CefSharp/wiki/General-Usage#win32-out-of-memory)
